content/ruby/concepts/trufflerruby/trufflerruby.md

```diff
@@ -0,0 +1,41 @@
+# Introduction

```

Comment:
There needs to be a metadata section. You can see how to build one in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).

You also need to remove the h1 header, it will be provided by the title given in the metadata.

Reply:
Hi Thank you so much for all of the enlightening comments,
Please check again the last commit include the added metadata and all other requested changes are done